xilem_web/README.md

```diff
@@ -57,18 +57,6 @@ This version of Xilem Web has been verified to compile with **Rust 1.88** and la
 Future versions of Xilem Web might increase the Rust version requirement.
 It will not be treated as a breaking change and as such can even happen with small patch releases.
 
-<details>
-<summary>Click here if compiling fails.</summary>
-
-As time has passed, some of Xilem Web's dependencies could have released versions with a higher Rust requirement.
-If you encounter a compilation issue due to a dependency and don't want to upgrade your Rust toolchain, then you could downgrade the dependency.
-
-```sh
-# Use the problematic dependency's name and version
-cargo update -p package_name --precise 0.1.1
-```
-</details>
-
 <!-- We hide these elements when viewing in Rustdoc, because they're not expected to be present in crate level docs -->
 <div class="rustdoc-hidden">
```

Comment:
👀

Reply:
I think that one is normal. We don't do `cargo rdme` for xilem_web. Hold on, let me check...

Reply:
Yup, we don't.